src/rules/prefer-flowmax.coffee

```diff
@@ -0,0 +1,68 @@
+{isFlow, isMagic, isBranchPure, getFlowToFlowMaxFixer, isNonmagicHelper, isFunction} = require '../util'
+
+module.exports =
+  meta:
+    docs:
+      description: 'Flag flow() calls that could be converted to flowMax()'
+      category: 'Possible Errors'
+      recommended: yes
+    schema: [
+      enum: ['always', 'whenUsingUnknownHelpers']
+    ,
+      type: 'object'
+      properties:
+        shouldFix:
+          type: 'boolean'
+        whitelist:
```

Comment:
Added `whitelist` so that you can explicitly designate certain helpers as nonmagic